src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -27,64 +26,59 @@ const defaultProps = {
     onfidoToken: null,
 };
 
-class RequestorOnfidoStep extends React.Component {
-    constructor(props) {
-        super(props);
-        this.submit = this.submit.bind(this);
-    }
+const stepCounter = {step: 3, total: 5};
 
-    submit(onfidoData) {
-        BankAccounts.verifyIdentityForBankAccount(lodashGet(this.props.reimbursementAccount, 'achData.bankAccountID') || 0, onfidoData);
+function RequestorOnfidoStep({onBackButtonPress, reimbursementAccount, onfidoToken}) {
+    const {translate} = useLocalize();
 
+    const submitOnfidoData = (onfidoData) => {
+        BankAccounts.verifyIdentityForBankAccount(lodashGet(reimbursementAccount, 'achData.bankAccountID') || 0, onfidoData);
         BankAccounts.updateReimbursementAccountDraft({isOnfidoSetupComplete: true});
-    }
+    };
 
-    render() {
-        return (
-            <ScreenWrapper
-                includeSafeAreaPaddingBottom={false}
-                shouldShowOfflineIndicator={false}
-            >
-                <HeaderWithBackButton
-                    title={this.props.translate('requestorStep.headerTitle')}
-                    stepCounter={{step: 3, total: 5}}
-                    shouldShowGetAssistanceButton
-                    guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
-                    onBackButtonPress={this.props.onBackButtonPress}
-                />
-                <FullPageOfflineBlockingView>
-                    <ScrollView contentContainerStyle={styles.flex1}>
-                        <Onfido
-                            sdkToken={this.props.onfidoToken}
-                            onUserExit={() => {
-                                BankAccounts.clearOnfidoToken();
-                                BankAccounts.goToWithdrawalAccountSetupStep(CONST.BANK_ACCOUNT.STEP.REQUESTOR);
-                            }}
-                            onError={() => {
-                                // In case of any unexpected error we log it to the server, show a growl, and return the user back to the requestor step so they can try again.
-                                Growl.error(this.props.translate('onfidoStep.genericError'), 10000);
-                                BankAccounts.clearOnfidoToken();
-                                BankAccounts.goToWithdrawalAccountSetupStep(CONST.BANK_ACCOUNT.STEP.REQUESTOR);
-                            }}
-                            onSuccess={(onfidoData) => {
-                                this.submit(onfidoData);
-                            }}
-                        />
-                    </ScrollView>
-                </FullPageOfflineBlockingView>
-            </ScreenWrapper>
-        );
-    }
+    const handleOnfidoError = () => {
+        // In case of any unexpected error we log it to the server, show a growl, and return the user back to the requestor step so they can try again.
+        Growl.error(translate('onfidoStep.genericError'), 10000);
```

Comment:
It is optional, but it is better to avoid "magic-numbers", you can create one more constant near `STEP_COUNTER`.
```suggestion
        Growl.error(translate('onfidoStep.genericError'), ERROR_TIMEOUT);
```